app/serializers/api/v3/serializable_course.rb

```diff
@@ -19,7 +19,7 @@ def enrichment_attribute(name, enrichment_name = name)
                  :content_status, :ucas_status, :funding_type,
                  :level, :is_send?, :english, :maths, :science, :gcse_subjects_required,
                  :age_range_in_years, :accrediting_provider,
-                 :accrediting_provider_code, :level
+                 :accrediting_provider_code, :level, :updated_at
```

Comment:
I think we need ‘changed_at’ instead as that looks at all associations too

Reply:
I wasn't aware of this. It has now been changed :)